java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/utils/Json.java

```diff
@@ -18,89 +18,207 @@
 
   private final JSONObject document;
 
+  /**
+   * Creates a new {@code Json} instance.
```

Comment:
Can describe the purpose of this class. That is a wrapper of the actual JSONObject, but adding helper methods to suit the SDK needs. 

Can add the description here, or at the top of the class.